docs/customizing.rst

```diff
@@ -0,0 +1,81 @@
+Customizing
+===========
```

Comment:
What do you think about reordering this page in order of most important (or likely to need to be changed) to least? I'm thinking...

1. Project Name & Logo
2. Header & Footer Links
3. GitHub Link
4. Show Date Last Updated
5. Show Copyright in Footer

Reply:
Rearranged in about that order (github link after project name)